packages/integrations/components/pico/useWidgetScript.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable max-len */
+import { useEffect, useCallback } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import getLogService from '@irvingjs/services/logService';
+import { actionPicoScriptAdded } from '../../actions/picoActions';
+import { picoScriptAddedSelector } from '../../selectors/picoSelector';
+
+const log = getLogService('irving:pico');
+
+export default function useWidgetScript(widgetUrl, publisherId) {
```

Comment:
Seems like this could be a cool thing to abstract at some point too (functionality for adding an inline script, unlike `useLoadScript` which adds a script tag with `src`)